pipelines/nextflow/workflows/nextflow.config

```diff
@@ -88,4 +95,8 @@ process {
         errorStrategy = { task.exitStatus in [104, 137, 140, 143] ? 'retry' : 'finish' }
         maxRetries = 3
     }
+    withLabel: 'cached' {
```

Comment:
This needs to be added to the tests nextflow config as well, otherwise the pipelines tests are broken since they can't use the cached data that we've prepared.

Alternatively, put this into a common nextflow file, and load it in both this file and the test file.

Reply:
NB: I believe this is the main remaining blocking part for the pipeline tests to run correctly

Reply:
I wonder if you can "redefine" labels, i.e. you can have a `nextflow.config` in the tests area that loads this one first, and then changes the content of those labels again here (so for instance things like `retry_on_error` (if introduced) and `cached` could stay as they are).